apps/wizard/app_pages/chart_diff/show.py

```diff
@@ -319,6 +433,42 @@ def show(self):
         else:
             self._show()
 
+        self.clean_cache()
+
+
+def compare_dictionaries(dix_1: Dict[str, Any], dix_2: Dict[str, Any]):
```

Comment:
We have tons of useful "diff"-like functions scattered across our repo (for dataframes, dicts, yamls, etc.). One day we can unify it into a single library (if it doesn't turn out to be too bespoke).

Reply:
totally;